packages/web-app-files/src/components/Collaborators/Collaborator.vue

```diff
@@ -357,7 +361,24 @@ export default {
 }
 </script>
 
-<style scoped="scoped">
+<style lang="scss" scoped="scoped">
+#collaborator-list {
```

Comment:
What do you think about using our margin and padding utility classes on the respective elements instead of explicitly defining margins and paddings here?

Reply:
Great idea, implemented